### PR TITLE
Update CronJob log generator to take namespace parameters.

### LINKED ISSUE #443: Update CronJob log generator to take a namespace parameters
The `CronJob` log generator needs to be updated to take a namespace parameter.

### PR DESCRIPTION
**Pull request by ggallen**
## Which Issue(s) this Pull Request Resolves

Resolves #443

## Release Note

```release-note
NONE
```

### CODE REVIEW

File: test/log-generators/cronjobs/install.sh [hunk 1]
```diff
@@ -1,9 +1,48 @@
...
+
+HELP=${HELP:-""}
+UNKNOWN=${UNKNOWN:-""}
+export NAMESPACE=${NAMESPACE:-generate-logs-cronjobs}
+
+# Help and usage
+if [ "${HELP}" == "True" ] || [ "${UNKNOWN}" == "True" ]; then
+    echo -e "$0
+
+    --namespace    Namespace to use to log generation.
```
Comment (beerparty) [inline test/log-generators/cronjobs/install.sh:35]: Do you mean "...use to generate logs"?

```suggestion
    --namespace    Namespace to use to generate logs.
```

File: test/log-generators/cronjobs/install.sh [hunk 2]
```diff
@@ -1,9 +1,48 @@
...
+    "
+    if [ "${UNKNOWN}" == "True" ]; then
+      exit 1;
+    else
+      exit 0;
+    fi
+fi
+
+
+kubectl create ns ${NAMESPACE} --dry-run=client -o yaml | kubectl apply -f -
```
Comment (beerparty) [inline test/log-generators/cronjobs/install.sh:47]: Why are you using the "--dry-run" flag to generate a YAML stream only to immediately apply the YAML stream? Why not just run "kubectl create ns ${NAMESPACE}"?

**Review by beerparty — CHANGES_REQUESTED** [review]

File: test/log-generators/cronjobs/install.sh (hunk 1 above)
Comment (ggallen) [reply test/log-generators/cronjobs/install.sh]: I meant "for log generation", but your phrasing works too.

Fixed.

File: test/log-generators/cronjobs/install.sh (hunk 2 above)
Comment (beerparty) [reply test/log-generators/cronjobs/install.sh:47]: Okay, that makes sense. Thanks.

**Review by beerparty — APPROVED** [review]

**Pull request merged**